Documentation/source/guide/tutorials-imagesequence.rst

```diff
@@ -0,0 +1,209 @@
+~~~~~~~~~~~
+For Blender VSE, (go to render tab>file format FFmpeg Video>your preferred codec and container)
+
+Adobe Premiere
+~~~~~~~~~~~~~~
+Import the image "as sequence" in the timeline and render in your preferred video format. 
+
+
+Creating Digital Intermediate For Editing Servers
+=================================================
+
+For Digital Intermediate, the `PRORES 4444` codec is a nice choice for MOV containers. It supports 12-bit with YUVA and retains alpha with 16-bit precision.
+
+It can be done with ffmpeg or in kdenlive/Shotcut importing the TIFF/PNG as sequence.
+
+
+FFmpeg
+~~~~~~
+``ffmpeg -framerate 30 -i input%03d.tiff -f mov -acodec pcm_s16le -vcodec prores_ks -vprofile 4444 -vendor ap10 -pix_fmt yuva444p10le out.mov``
+
+Shotcut/Kdenlive
+~~~~~~~~~~~~~~~~
+- A render profile needs to be created first with below profile:
+- ``f=mov acodec=pcm_s16le vcodec=prores_ks vprofile=4444 vendor=ap10 pix_fmt=yuva444p10le qscale=%quality``
+- Use TIFF/PNG image as sequence in the timeline.
+- Then Render with this newly created prores 4444 profile.
+
+A Tutorial on PRORES in LINUX by CGVIRUS:
+https://youtu.be/oBiaBYthZSo
+
+It can be done with Adobe Premiere/avid/fcpx/resolve etc as well by importing TIFF/PNG as sequence and render as MOV prores 4444.
+
+DaVinci Resolve, Adobe Premiere etc
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Drag and drop the folder containing the image sequence to a timeline.
+- Render the timeline in PRORES 4444.
+
+How to Render Image Sequences from Natron
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The preferred file format to render out composited frames is TIFF.(image attached):
+
+In the write node:
+
+- output components can be RGB(no transparency) or RGBA(with transparency)
+<<<<<<< HEAD
+- Use filename_###.tiff (where # is the frame number and padding) ### will create yourfilename001.tiff and ## will create yourfilename01.tiff
+- Bit depth can be auto/8i/16i (Don't use float)
+- compression can be none/lzw (HD). for 4k deflate is ok.
+
+.. figure:: _images/imagesequence_1.jpg
+=======
+- Use filename_###.tiff (where # is the frame number and padding) 
+### will create yourfilename001.tiff and ## will create yourfilename01.tiff
+- Bit depth can be auto/8i/16i (Don't use float)
+- compression can be none/lzw (HD). for 4k deflate is ok.
+>>>>>>> cd1aa5ea7a99d4fa4bf0dec8f0fcf660a3757888
+
+PNG is also a good format:
+
+In the write node:
+
+- output components can be RGB(no transparency) or RGBA(with transparency)
+- Use filename###.png (where # is the frame number and padding) 
+### will create yourfilename001.png and ## will create yourfilename01.png
+- Bit depth can be 8/16bit
+- compression can be 0 for HD, 6 for 4k is fair enough.
+<<<<<<< HEAD
+=======
+
+>>>>>>> cd1aa5ea7a99d4fa4bf0dec8f0fcf660a3757888
+
+.. figure:: _images/imagesequence_2.jpg
+
+
+Open Questions for this document:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+What format should I use for frames? (esp if the video is 10bit or 12bit) ?
```

Comment:
I'm usually told to use EXR.

Reply:
Do you have a reliable source for that? It would be good to link it to somewhere official.

Reply:
> Do you have a reliable source for that? It would be good to link it to somewhere official.

https://www.openexr.com/about.html
On that page, they said "8- and 10-bit formats lack the dynamic range necessary to store high-contrast images captured from HDR devices." That is one of the reasons why EXR exist.

https://www.openexr.com/index.html
"High dynamic range and color precision."

Reply:
"Weta Digital, Walt Disney Animation Studios, Sony Pictures Imageworks, Pixar Animation Studios, DreamWorks, and other studios, companies, and individuals have made significant contributions to the code base."

These names can make EXR format more "standard".